[2023-02-23T18:45:24.852+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: metadata_scraper.scrape_nexrad_data manual__2023-02-23T18:45:21.211353+00:00 [queued]>
[2023-02-23T18:45:24.862+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: metadata_scraper.scrape_nexrad_data manual__2023-02-23T18:45:21.211353+00:00 [queued]>
[2023-02-23T18:45:24.864+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-23T18:45:24.866+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-02-23T18:45:24.867+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-23T18:45:24.879+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): scrape_nexrad_data> on 2023-02-23 18:45:21.211353+00:00
[2023-02-23T18:45:24.898+0000] {standard_task_runner.py:55} INFO - Started process 1060 to run task
[2023-02-23T18:45:24.910+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'metadata_scraper', 'scrape_nexrad_data', 'manual__2023-02-23T18:45:21.211353+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/metadataScrapper.py', '--cfg-path', '/tmp/tmpbflnjg1m']
[2023-02-23T18:45:24.915+0000] {standard_task_runner.py:83} INFO - Job 81: Subtask scrape_nexrad_data
[2023-02-23T18:45:24.999+0000] {task_command.py:388} INFO - Running <TaskInstance: metadata_scraper.scrape_nexrad_data manual__2023-02-23T18:45:21.211353+00:00 [running]> on host a63d292ab6ab
[2023-02-23T18:45:25.062+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Charu Singh
AIRFLOW_CTX_DAG_ID=metadata_scraper
AIRFLOW_CTX_TASK_ID=scrape_nexrad_data
AIRFLOW_CTX_EXECUTION_DATE=2023-02-23T18:45:21.211353+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-23T18:45:21.211353+00:00
[2023-02-23T18:45:25.069+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/metadataScrapper.py", line 93, in nexrad_data
    nexrad_bucket), Prefix=prefix, Delimiter='/')
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/client.py", line 530, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/client.py", line 924, in _make_api_call
    headers=additional_headers,
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/client.py", line 988, in _convert_to_request_dict
    api_params, operation_model, context
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/client.py", line 1030, in _emit_api_params
    context=context,
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/hooks.py", line 412, in emit
    return self._emitter.emit(aliased_event_name, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/hooks.py", line 256, in emit
    return self._emit(event_name, kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/hooks.py", line 239, in _emit
    response = handler(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/botocore/handlers.py", line 279, in validate_bucket_name
    if not VALID_BUCKET.search(bucket) and not VALID_S3_ARN.search(bucket):
TypeError: expected string or bytes-like object
[2023-02-23T18:45:25.080+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=metadata_scraper, task_id=scrape_nexrad_data, execution_date=20230223T184521, start_date=20230223T184524, end_date=20230223T184525
[2023-02-23T18:45:25.089+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 81 for task scrape_nexrad_data (expected string or bytes-like object; 1060)
[2023-02-23T18:45:25.120+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-02-23T18:45:25.156+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
